### PR TITLE
Feat/better prone errors

### PR DESCRIPTION
**Pull request by ktfth**
All needed to investigate a process flow in pura with instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,12 +44,34 @@ cargo build --release
 
 **Pura** can be used as a standalone container runtime like **runc** with the OCI compliant commands:
 
+Make this to clean the way:
+```sh
+set ID=example
+[sudo] unlink /tmp/pura/${ID}/init.sock
+[sudo] unlink /tmp/pura/${ID}/container.sock
```
Comment (penumbra23) [inline README.md:51]: This shouldn't be in the README as it's a bug in the runtime with the cleanup. You can make a new file `NOTES.md` for internal contributor notes and add these comments. Same for the `netcat` command below.

File: README.md [hunk 2]
```diff
@@ -44,12 +44,34 @@ cargo build --release
...
+```sh
+[sudo] nc -vklU /tmp/${ID}.sock # on another terminal
+```
+
+On the config.json
+```sh
+...
+"root": {
+  "path": "/path/to/rootfs"
+}
```
Comment (penumbra23) [inline README.md:64]: Actually, the `root` section on Linux is required, so we don't need to specify it in the documentation.

File: README.md [hunk 3]
```diff
@@ -44,12 +44,34 @@ cargo build --release
...
+```
+
+Now on pura context
+```sh
 cd target/release
-./pura create id123456789 --bundle /path/to/bundle
-./pura start id123456789
-./pura state id123456789
-./pura delete id123456789
+./pura create ${ID} --bundle /path/to/bundle --console-sock /tmp/${ID}.sock
```
Comment (penumbra23) [inline README.md:71]: ` --console-sock` option is required just in the case when `process.terminal` in the config.json is set to true. It's not required for running container.

File: README.md (hunk 2 above)
Comment (ktfth) [reply README.md]: You need to specific the complete path to `rootfs` directory

Comment (penumbra23) [reply README.md]: Yup, got it this morning in my tests.

**Review by penumbra23 — APPROVED** [review]
Almost the same error msg injection I've used on some branch. Thanks!

**Comment by penumbra23** [comment]
@ktfth 👍

**Pull request merged**